### PR TITLE
feat: add week start day to calendar widget

### PR DESCRIPTION
**Pull request by hillaliy**
<br/>
<div align="center">
  
  <h3>Homarr</h3>
</div>

**Thank you for your contribution. Please ensure that your pull request meets the following pull request:**

- [x] Builds without warnings or errors (``pnpm buid``, autofix with ``pnpm format:fix``)
- [x] Pull request targets ``dev`` branch
- [x] Commits follow the [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] No shorthand variable names are used (eg. ``x``, ``y``, ``i`` or any abbrevation)

### CODE REVIEW

File: packages/translation/src/lang/en.ts [hunk 1]
```diff
@@ -1009,6 +1009,9 @@ export default {
       name: "Calendar",
       description: "Display events from your integrations in a calendar view within a certain relative time period",
       option: {
+        weekStartOnSanday: {
+          label: "Week start on sanday",
```
Comment (SeDemal) [inline packages/translation/src/lang/en.ts:1013]: typo: "sunday"

**Review by SeDemal — CHANGES_REQUESTED** [review]

**Comment by Meierschlumpf** [comment]
Bro :joy: I worked at exactly the same thing, but rather than putting it into the calendar I put it in user settings :joy:
What's your opinion on that, should this be user specific or defined in the widget itself?

**Comment by SeDemal** [comment]
> Bro 😂 I worked at exactly the same thing, but rather than putting it into the calendar I put it in user settings 😂 What's your opinion on that, should this be user specific or defined in the widget itself?
> 
> Mine would be #1249

Omg I thought I would have thought of that when I made the widget, so that's why it wasn't in already.
I think, either you do it for both the clock and the calendar, or we leave it both in options

**Comment by SeDemal** [comment]
> > Omg I thought I would have thought of that when I made the widget, so that's why it wasn't in already. I think, either you do it for both the clock and the calendar, or we leave it both in options
> 
> What do you mean in the clock? Is there something related to first day of week?


time representation with 24h format or am/pm, date order between month and day.
All of them are localizations proper to the user in the end.

**Comment by Meierschlumpf** [comment]
Maybe we can create a seperate issue for that? I think it's okay as we have it right now for now

**Comment by SeDemal** [comment]
Yeah we can make a different issue/PR for that, but to me it's the same, a representation of time that is proper to a user's preference.
Overall though, making it a user setting and not a board setting seems like a good idea.
@manuel-rw thoughts?

**Comment by hillaliy** [comment]
Ok my friend, on settings it will be better. I'll close this PR

**Pull request closed without merging**